docs/content/direct/core-chart-argocd.md

```diff
@@ -22,12 +32,97 @@ When deploying in a **Kubernetes** cluster, use the flag `--set argocd.global.do
+```
+
+**Expected output (similar to):**
+```
+Release "ks-core" has been upgraded. Happy Helming!
+NAME: ks-core
+LAST DEPLOYED: Tue Jun  3 11:28:26 2025
+NAMESPACE: default
+STATUS: deployed
+REVISION: 3
+TEST SUITE: None
+NOTES:
+For your convenience you will probably want to add contexts to your
+kubeconfig named after the non-host-type control planes (WDSes and
+ITSes) that you just created (a host-type control plane is just an
+alias for the KubeFlex hosting cluster). You can do that with the
+following `kflex` commands; each creates a context and makes it the
+current one. See
+https://github.com/kubestellar/kubestellar/blob/0.28.0-alpha.2/docs/content/direct/core-chart.md#kubeconfig-files-and-contexts-for-control-planes
+for a way to do this without using `kflex`.
+Start by setting your current kubeconfig context to the one you used
+when installing this chart.
+
+kubectl config use-context $the_one_where_you_installed_this_chart
+kflex ctx --set-current-for-hosting # make sure the KubeFlex CLI's hidden state is right for what the Helm chart just did
+
+Finally, you can use `kflex ctx` to switch back to the kubeconfig
+context for your KubeFlex hosting cluster.
+
+Access Argo CD UI at https://argocd.localtest.me (append :9443 for Kind or k3s installations).
+Obtain Argo CD admin password using the command:
+kubectl get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
+```
+
+**Verify Argo CD installation:**
+
+```bash
+kubectl get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
+```
+
+**Expected output (similar to):**
+```
+SYttSMZy6TQ2-alF
+```
+
+```bash
+kubectl get pods -A | grep -i argo
+```
+
+**Expected output (similar to):**
+```
+default              ks-core-argocd-application-controller-0                     1/1     Running     0          15m
+default              ks-core-argocd-applicationset-controller-6669c9f789-wd5h7   1/1     Running     0          15m
+default              ks-core-argocd-dex-server-8464bc64b9-dplv5                  1/1     Running     0          15m
+default              ks-core-argocd-notifications-controller-66b8ccc4c7-8mjdx    1/1     Running     0          15m
+default              ks-core-argocd-redis-76c6b4db57-7cfrs                       1/1     Running     0          15m
+default              ks-core-argocd-repo-server-6774bd65db-rxmtz                 1/1     Running     0          15m
+default              ks-core-argocd-server-84cbbd8cbd-bpl92                      1/1     Running     0          15m
+```
+
+**Access Argo CD UI:**
+
+Open your browser and navigate to: `https://argocd.localtest.me:9443/`
+
+- **Username**: `admin`
+- **Password**: Use the password obtained from the previous command (e.g., `SYttSMZy6TQ2-alF`)
+
+> **Note:** If you encounter SSL certificate warnings in your browser, proceed with "Advanced" → "Proceed to argocd.localtest.me (unsafe)" or similar option, as this is expected for local development setups.

```

Comment:
I think that it would be good to be more specific about which browser warning this is trying to tell the reader to brush past --- it is possible that a user has something else go wrong! This is specifically about the warning about self-signed server certificate, right?

Reply:
There can be certificate problems other than self-signed certificates. If I understand correctly, this text is trying to tell the reader to push past the warning about a self-signed certificate. If I have this right then a more specific note is warranted.